Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1462,7 +1463,7 @@ private async Task<AuthenticationResult> AuthenticateNewSessionInternal(Authenti
 
             if (user is null)
             {
-                await _eventManager.PublishAsync(new GenericEventArgs<AuthenticationRequest>(request)).ConfigureAwait(false);
+                await _eventManager.PublishAsync(new GenericEventArgs<AuthenticationRequestEventArgs>(new AuthenticationRequestEventArgs(request))).ConfigureAwait(false);
```

Comment:
Wouldn't it be better to make `AuthenticationRequestEventArgs` extend `EventArgs` instead of wrapping it in `GenericEventArgs`?